ondivi/entry.py

```diff
@@ -97,7 +97,7 @@ def cli(
         else sys.stdin.read().strip().splitlines()
     )
     try:
-        diff = Repo('.').git.diff('--unified=0', baseline)
+        diff = Repo('.').git.diff('--unified=0 --no-ext-diff', baseline)
```

Comment:
```suggestion
        diff = Repo('.').git.diff('--unified=0', '--no-ext-diff', baseline)
```
Unfortunately I believe that putting multiple arguments into a single string will not work. I believe you'll need to separate them into multiple strings.

Reply:
Confirmed: 

```python
>>> from git import Repo
>>> a = Repo('.')
>>> a.git.diff('--unified=0', '--no-ext-diff', 'main')
'diff --git a/file2 b/file2\nnew file mode 100644\nindex 0000000..e69de29'
>>> a.git.diff('--unified=0', 'main')
'diff --git a/file2 b/file2\nnew file mode 100644\nindex 0000000..e69de29'
>>> a.git.diff('--unified=0 --no-ext-diff', 'main')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/markdelcambre/test/.venv/lib/python3.12/site-packages/git/cmd.py", line 986, in <lambda>
    return lambda *args, **kwargs: self._call_process(name, *args, **kwargs)
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/markdelcambre/test/.venv/lib/python3.12/site-packages/git/cmd.py", line 1599, in _call_process
    return self.execute(call, **exec_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/markdelcambre/test/.venv/lib/python3.12/site-packages/git/cmd.py", line 1389, in execute
    raise GitCommandError(redacted_command, status, stderr_value, stdout_value)
git.exc.GitCommandError: Cmd('git') failed due to: exit code(129)
  cmdline: git diff --unified=0 --no-ext-diff main
  stderr: 'error: --unified expects a numerical value'